ui/src/buckets/actions/thunks.ts

```diff
@@ -55,17 +56,22 @@ export const getBuckets = () => async (
   getState: GetState
 ) => {
   try {
-    dispatch(setBuckets(RemoteDataState.Loading))
-    const org = getOrg(getState())
+    const state = getState()
+    if (getStatus(state, ResourceType.Buckets) === RemoteDataState.NotStarted) {
+      dispatch(setBuckets(RemoteDataState.Loading))
+    }
+    const org = getOrg(state)
 
-    const resp = await api.getBuckets({query: {orgID: org.id}})
+    const resp = await api.getBuckets({query: {}})
 
     if (resp.status !== 200) {
       throw new Error(resp.data.message)
     }
 
+    const bucketsInOrg = resp.data.buckets.filter(b => b.orgID === org.id)
```

Comment:
the idea is to expand this once the sample data org is created to hardcode another org id?

Reply:
when I can add sampledata type bucket
the filter will be `b.orgID === org.id || b.type==="sampledata"`